app/components/enter-hours-form.tsx

```diff
@@ -0,0 +1,224 @@
+"use client"
+
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card"
+import { ScrollArea } from "@radix-ui/react-scroll-area"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { useForm } from "react-hook-form"
+import { Button } from "@/components/ui/button"
+import * as z from "zod"
+import {
+    Form,
+    FormControl,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form"
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from "@/components/ui/select"
+
+export default function EnterHoursForm() {
+
+    const enterHoursSchema = z.object({
+        // should we just give the DOW to them? maybe in a cute disabled button or something? 
```

Comment:
They would need to enter their day of week won't they?

Reply:
![image](https://github.com/troypoulter/toil-calculator/assets/19419349/1fe70b35-1356-4cdd-8613-1c094957ea57)